src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = Lottery.getRandomBall();
+        Ball ball2 = Lottery.getRandomBall();
+        Ball ball3 = Lottery.getRandomBall();
+
+        System.out.println("Number : " + ball1.getNumbers() + ", " + "Color : " + ball1.getColor());
+        System.out.println("Number : " + ball2.getNumbers() + ", " + "Color : " + ball2.getColor());
+        System.out.println("Number : " + ball3.getNumbers() + ", " + "Color : " + ball3.getColor());
```

Comment:
1. Please use cycle for repetitive actions
2. You have prover override of Ball.toSting() method, use it